docs/src/Pulsar.doc.js

```diff
@@ -78,9 +78,7 @@ class FlyoutExample extends React.Component {
             top: 10,
             left: 10,
             pointerEvents: "none", }}>
-            <TapArea onTap={({ event }) => this.handleClick(event)} rounding="circle" fullWidth={false}>
               <Pulsar paused={this.state.open} />
-            </TapArea>
```

Comment:
@christianvuerings I'm unclear about the purpose of this TapArea here - happy to put it back if it's needed

Reply:
@ayeshakmaz no need for it, this should be handled by the `<Button />`